doc/user/content/overview/_index.md

```diff
@@ -10,185 +10,74 @@ menu:
+where data must be processed at speeds and scales not possible in traditional
+databases, but without the cost, complexity, and development time of most
+streaming engines.
+
+If you're looking to reduce the time it takes to refresh answers to common
+queries, Materialize can make that happen! Rather than recalculating results
+from scratch each time it's queried, or serving stale cached results,
+Materialize continually ingests data and keeps results up to date for you
+as **incrementally updated materialized views**.
+
+## Features
+
+Materialize combines all the things you love about databases and abstracts all
+the hard parts that go into making a streaming engine horizontally scalable,
+highly available and strongly consistent. To get up and running, all you need
+to know is **SQL**.
+
+### Incrementally updated views
+
+Materialized views are useful to avoid re-running heavy queries that are asked
+over and over against your database, and typically use caching to serve results
+faster. In traditional databases, materialized views often come with a long
+list of limitations and force you to make a compromise between the freshness of
+the results, the cost of refreshing the view and the complexity of the SQL
+statements you can use.
+
+Materialize supports incrementally updated materialized views that are **always
+fresh**, and allows a much broader set of SQL statements, like views over
+multi-way joins with complex aggregations. How? Under the hood, its engine is
+powered by [Timely](https://github.com/TimelyDataflow/timely-dataflow#timely-dataflow) and
 [Differential Dataflow](https://github.com/timelydataflow/differential-dataflow#differential-dataflow),
-purpose-built for low-latency applications. It lets you ask complex questions
-about your data using **SQL**, and incrementally maintains the results of these SQL queries as the underlying data changes.
+backed by many years of research and optimized for this exact purpose. Best
+yet: as a user, you don't really need to think about it — it just works™️.
 
-## What does Materialize do?
+### Standard SQL support
 
-You might be wondering: why not just use your database's built-in functionality
-to perform these same computations? Your database often acts as if it's never
-been asked that question before, which means it can take a _long_ time to come
-up with an answer, each and every time you pose the query.
+### PostgreSQL wire-compatibility
 
-Materialize instead keeps the results of the queries and incrementally updates
-them as new data comes in. So, rather than recalculating the answer each time
-it's asked, Materialize continually updates the answer and gives you the
-answer's current state.
+Every database needs a protocol to standardize communication with the outside
+world. Materialize uses the [PostgreSQL wire protocol](https://datastation.multiprocess.io/blog/2022-02-08-the-world-of-postgresql-wire-compatibility.html),
```

Comment:
Our [explainer](https://materialize.com/blog/postgres-compatibility/#1-postgresql-wire-compatibility) is very good, but it doesn't dig deeper into what it means to be wire-compatible with PostgreSQL. I find this blogpost from Phil Eaton very useful to understand the idea! 

This reminds me that we should  consider #8086 in #18508, @jseldess.

Reply:
Thanks, @morsapaes. Added a comment here: https://github.com/MaterializeInc/materialize/issues/8086#issuecomment-1525770117.